tests/shared/utils/test_common.py

```diff
@@ -0,0 +1,14 @@
+import random
+
+import rasa.shared.utils.common
+
+
+def test_all_subclasses():
+    num = random.randint(1, 10)
+
+    class TestClass:
+        pass
+
+    classes = [type(f"TestClass{i}", (TestClass,), {}) for i in range(num)]
```

Comment:
it doesn't really test for sub-subclasses and doesn't check that if a class is not a subclass, then it's not included

(i know you haven't introduced the code, feel free to add a FIXME if lack of time)

Reply:
> it doesn't really test for sub-subclasses 

It does actually create subclasses (https://docs.python.org/3/library/functions.html#type)

>  doesn't check that if a class is not a subclass, then it's not included

What would you like to test here? The fact that other, random classes, aren't found by this function?

Reply:
I meant sub-sub classes so that uses the recursion

Reply:
will address that in a separate PR 👍 (wanted to get this merged)